Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/storage/entity/TraceLocationEntity.kt

```diff
@@ -18,8 +18,8 @@ data class TraceLocationEntity(
     @ColumnInfo(name = "startDate") val startDate: Instant?,
     @ColumnInfo(name = "endDate") val endDate: Instant?,
     @ColumnInfo(name = "defaultCheckInLengthInMinutes") val defaultCheckInLengthInMinutes: Int?,
+    @ColumnInfo(name = "byteRepresentation") val byteRepresentationBase64: String,
```

Comment:
Should the columnName then also be `byteRepresentationBase64` ?

Reply:
Yes makes sense. However we are also not suffixing the base64 encoded values in the `CheckInEntity`. I think a PR in which we refactor the 2 entities would be best. I can do this after this one is merged, if that's fine for you?

Reply:
Sounds good.